src/modules/85-cv/pages/health-source/connectors/CommonHealthSource/__tests__/CommonHealthSourceContainer.test.tsx

```diff
@@ -128,11 +119,36 @@ describe('Unit tests for CommonHealthSourceContainer', () => {
       sli: false
     })
 
-    expect(createHealthSourceConfigurationsData(mockedSourceData, isTemplate)).toEqual({
+    expect(createHealthSourceConfigurationsData(mockedSourceData)).toEqual({
       customMetricsMap,
       failFastThresholds: [],
       ignoreThresholds: [],
       selectedMetric: 'M1'
     })
   })
+
+  test('should create correct data to to support edit health source', () => {
```

Comment:
should create correct data **to** to support edit health source
```suggestion
  test('should create correct data to support edit health source', () => {
```

Reply:
Done